style.css

```diff
@@ -65,8 +61,7 @@ body {
     overflow: hidden;
     position: relative;
     margin : 0 auto;
-    top : 20px;
-    //margin-top : -50px;
+    top : 10px;
```

Comment:
just merge! and lets move on to next version of the game of adding more features.